layer/USAGE_desktop.md

```diff
@@ -1,106 +1,141 @@
-  * Indirectly by using the
-`VK_INSTANCE_LAYERS` environment variable.
+After `VK_LAYER_PATH` has been updated, the layer may be enabled through
+one of the following methods:
+  * Adding the `VK_LAYER_LUNARG_gfxreconstruct` name string to the `VkInstanceCreateInfo::ppEnabledLayerNames` value when calling `vkCreateInstance`
+  * Adding the `VK_LAYER_LUNARG_gfxreconstruct` name string to the list of layer names specified through the `VK_INSTANCE_LAYERS` environment variable.
 
-### Setting the VK_LAYER_PATH
+### Setting VK_LAYER_PATH
+The `VK_LAYER_PATH` environment variable should be used to tell Vulkan loader
+where to find the GFXReconstruct layer.
 
 #### Windows
-
-If your source was located in: `C:\my_folder\gfxreconstruct`
-and your build folder was `build64`,
-then you would add it to the layer path in the following way:
-
+The following example demonstrates how to update the Windows `VK_LAYER_PATH`
+environment variable for the GFXReconstruct capture layer. The example uses
+the `C:\gfxreconstruct` path to represent the location of directory containing
+the GFXReconstruct project source, and the `build` folder name to represent
+the sub-directory specified to CMake as the location to place the build
+binaries. The following command would be executed from the command prompt to
+add the Debug build of the layer to `VK_LAYER_PATH`:
 ```
-set VK_LAYER_PATH=C:\my_folder\gfxreconstruct\build64\layer\Debug;%VK_LAYER_PATH%
+set VK_LAYER_PATH=C:\gfxreconstruct\build\layer\Debug;%VK_LAYER_PATH%
 ```
 
 #### Linux
-
-If your source was located in: `/my_folder/gfxreconstruct`
-and your build folder was `build`,
-then you would add it to the layer path in the following way:
-
+The following example demonstrates how to update the Windows `VK_LAYER_PATH`
+environment variable for the GFXReconstruct capture layer. The example uses
+the `/gfxreconstruct` path to represent the location of directory containing
+the GFXReconstruct project source, and the `build` folder name to represent
+the sub-directory specified to CMake as the location to place the build
+binaries. The following command would be executed from the command line to
+add the layer to `VK_LAYER_PATH`:
 ```
-export VK_LAYER_PATH=/my_folder/gfxreconstruct/build/layer;$VK_LAYER_PATH
+export VK_LAYER_PATH=/gfxreconstruct/build/layer;$VK_LAYER_PATH
 ```
 
-### Forcing the layer with VK_INSTANCE_LAYERS
-
-To force the layer to be enabled for Vulkan applications, you can
-set the `VK_INSTANCE_LAYERS` environment variable in the following way:
+### Enabling the layer with VK_INSTANCE_LAYERS
+The `VK_INSTANCE_LAYERS` environment variable may be used to force the Vulkan
+loader to load the GFXReconstruct layer.
 
 #### Windows
-
+The following command would be executed from the command prompt to set the
+`VK_INSTANCE_LAYERS` environment variable:
 ```
 set VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_gfxreconstruct
 ```
 
 #### Linux
-
+The following command would be executed from the command line to set the
```

Comment:
technically, from the `bash` command line... but I don't think this is a big deal